subgraph/src/metaBoard.ts

```diff
@@ -19,7 +19,17 @@ export function handleMetaV1_2(event: MetaV1Event): void {
   metaV1.sender = event.params.sender;
   metaV1.subject = event.params.subject;
 
-  metaV1.metaHash = MetaBoardContract.bind(event.address).hash(event.params.meta);
+  // Use `try_hash` to safely attempt to get the metaHash
+  let contract = MetaBoardContract.bind(event.address);
```

Comment:
there's actually a hash function in subgraph so we don't even need this

https://github.com/rainlanguage/rain.orderbook/blob/main/subgraph/src/clear.ts#L18